pytorch_lightning/core/lightning.py

```diff
@@ -1545,7 +1562,9 @@ def get_progress_bar_dict(self):
         if avg_training_loss is not None:
             tqdm_dict["loss"] = f"{avg_training_loss:.3g}"
 
-        if self.trainer.truncated_bptt_steps is not None:
+        module_tbptt_enabled = self.truncated_bptt_steps > 0
```

Comment:
Could we have collision there ? Maybe worth to use the same resolution as with `automatic_optimization`.